### ISSUE TITLE
IE 11 Support?

### ISSUE THREAD

**Issue by FrancescoFilippi** [GeoTIFF/georaster-layer-for-leaflet#19]
Hi @DanielJDufour , is there a way to make this work on Internet Explorer 11? That would be great!
Most of the problems I'm facing seem related to unsupported sintax.

**Comment by DanielJDufour** [comment]
Yes, I can definitely try!  To speed things up, would you be able to post a screenshot of the dev console with the error messages.  Thank you!

**Comment by FrancescoFilippi** [comment]
I'll definitely post screenshot as soon as I can. 
Meanwhile, I can recall at least three syntax issues:
- `async` function
https://github.com/GeoTIFF/georaster-layer-for-leaflet/blob/d32da7e9c554bc277da4d083be9bacaa42b87794/georaster-layer-for-leaflet.browserify.min.js#L186 low impact, the only await call is commented out
- Template literals
https://github.com/GeoTIFF/georaster-layer-for-leaflet/blob/d32da7e9c554bc277da4d083be9bacaa42b87794/utils/utm.js#L23

- Object literals
https://github.com/GeoTIFF/georaster-layer-for-leaflet/blob/d32da7e9c554bc277da4d083be9bacaa42b87794/georaster-layer-for-leaflet.browserify.min.js#L126
probably the most recurrent problem

Thank you!

**Comment by DanielJDufour** [comment]
These are still definitely issues.  Unfortunately, I haven't found the time to fix them.  @FrancescoFilippi , would you have time to work on this and submit a merge request?
